nuitka/tools/testing/Common.py

```diff
@@ -341,8 +343,11 @@ def compareWithCPython(dirname, filename, extra_flags, search_mode, needs_2to3):
     _removeCPythonTestSuiteDir()
 
     if result != 0 and result != 2 and search_mode.abortOnFinding(dirname, filename):
-        my_print("Error exit!", result)
-        sys.exit(result)
+        if search_mode.isSearchModeAll():
```

Comment:
I would rather make this `search_mode.onErrorDetected(message)` and not use a test method.

When I said that test methods of the `isXY` kind are better, that was true. But even better, make the search mode decide what's going to happen entirely, and just inform it about facts.

Reply:
Sorry, I don't understand. Do you want me to store and display errors instead of just incrementing the error counter?